docs/source/conf.py

```diff
@@ -43,8 +43,8 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-# This theme is the most popular one and the one used by the 
-# Read the Docs website https://docs.readthedocs.io/en/stable/index.html 
+# This theme is the most popular one and the one used by the
+# Read the Docs website https://docs.readthedocs.io/en/stable/index.html
```

Comment:
was this also a cause by a linting error? might be better to put linting fixes in a different PR

Reply:
yes - is there a way to disable linting errors for this PR so I can get this through without fixing the linting errors in this PR?

Reply:
@pmantica1 IIRC I think you checked in this code. Could you:
- check your local environment to make sure it is configured to strip whitespace at the end of lines, and
- investigate with @jecdohmann why the linter complained for him but allowed us to merge the original change in?

Reply:
Will do.

Reply:
```# Get all python files or directories that need to be linted.
lintable_locations="graphql_compiler/"
# pylint doesn't support linting directories that aren't packages:
# https://github.com/PyCQA/pylint/issues/352
# Use **/*.py to supply all python files for individual linting.
pylint_lintable_locations="graphql_compiler/**/*.py"
if [ "$diff_only" -eq 1 ] ; then
    # Quotes don't need to be escaped because they nest with $( ).
    lintable_locations="$(git diff --name-only master... | grep ".*\.py$")"
    pylint_lintable_locations="$lintable_locations"
fi```

The default value of `lintable_locations` is the `graphql_compiler/` directory. However, if you run `lint.sh` with the `--diff` parameter you can include files that are not in the `graphql_compiler` directory, such as those in readthedocs.

Reply:
Also I am not sure that the regex `"\.*\.py$"` is correct. Shouldn't it be ".*\.py$"? (Without the first slash).

Reply:
I think you might be right. Test it out to be sure and open a PR?

Reply:
Let's also make `docs` be a lintable directory if possible.

Reply:
Also `scripts`, it seems like that's not marked as lintable either. Thanks @jecdohmann for finding these issues with our linter setup, and sorry for the frustrations they must have caused.

Reply:
No worries - Pedro, let me know if you want to discuss this at all with me. Otherwise, I'll just leave the linting stuff as is for this PR